modules/local/validation/fileintegrity/main.nf

```diff
@@ -0,0 +1,152 @@
+process VALIDATION_FILEINTEGRITY {
```

Comment:
Perhaps we should rethink this approach, would be difficult down the line to incorporate other file-integrity steps [(e.g. what if we want to check vcf header)](https://vcftools.github.io/perl_module.html#vcf-validator)
Perhaps what we can do is, split channel per file. Run check per file according to type , join on meta and update status if all files pass.